src/components/Tabs/Tabs.tsx

```diff
@@ -12,15 +13,45 @@ interface ITabsProps {
 }
 
 export const Tabs = ({ children }: ITabsProps) => {
+  const [showScrollButton, setShowScrollButton] = useState(false);
+  const tabContent = document.getElementById('tab-content');
+
+  const onResize = useCallback(() => {
+    tabContent && setShowScrollButton(tabContent.scrollWidth > window.innerWidth);
+  }, [tabContent]);
+
+  window.addEventListener('resize', onResize);
+
+  const scrollLeft = () => {
+    if (tabContent) {
+      tabContent.scrollLeft = tabContent.scrollLeft ? tabContent.scrollLeft - 150 : 0;
+    }
+  };
+  const scrollRight = () => {
+    if (tabContent) {
+      tabContent.scrollLeft = tabContent.scrollLeft ? tabContent.scrollLeft + 150 : 150;
+    }
+  };
+
+  useEffect(onResize, [onResize]);
```

Comment:
I have several points:
1) you could avoid `onResize` in dependency array
2) you miss remove event listener
3) `getElementById` should be used after component is rendered, also check if `useRef` should be used instead?

I am providing some draft for you:
```tsx
useEffect(() => {
  const onResize = /* ... */ 

  tabContent = /* ... */
  window.addEventListener('resize', onResize);
  return () => {
    window.removeEventListener('resize', onResize);
  }
}, []);
```